Formula/elektra.rb

```diff
@@ -0,0 +1,48 @@
+class Elektra < Formula
+  desc "Configuration Framework"
+  homepage "https://libelektra.org"
+  url "https://www.libelektra.org/ftp/elektra/releases/elektra-0.8.21.tar.gz"
+  sha256 "51892570f18d1667d0da4d0908a091e41b41c20db9835765677109a3d150cd26"
+  head "https://github.com/ElektraInitiative/libelektra.git"
+
+  option "with-qt", "Build GUI frontend"
+
+  # Build Dependencies
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+
+  # Run-Time Dependencies
+  depends_on "qt" => :optional
+  depends_on "discount" if build.with? "qt"
+
+  def install
+    cmake_args = %W[
+      -DCMAKE_BUILD_TYPE=Release
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
+      -DBINDINGS='cpp'
+      -DTOOLS='kdb'
+      -DPLUGINS='NODEP'
+    ]
+
+    if build.with? "qt"
+      tools << "qt-gui"
```

Comment:
`tools` seems to be unused?

Reply:
You are right. Thank you for the hint. I totally forgot about this after I deleted large parts of the formula. Should be fixed now.